modules/tinymce/src/core/test/ts/browser/selection/GetSelectionContentTest.ts

```diff
@@ -15,6 +15,19 @@ describe('browser.tinymce.selection.GetSelectionContentTest', () => {
   }, []);
   const testDivId = 'testDiv1';
 
+  const assertSelectedRadioButtons = (editor: Editor, nrOfInputs: number, shouldBeSelected: number) => {
```

Comment:
It looks like the same `assertSelectedRadioButtons` function defined in the` EditorGetContentTextFormatTest.ts`. Maybe better to move it in some common place.

Reply:
It's the same, but I didn't feel it was worth setting up a set of helper-files for it, since it's just a single, small function.